features/storage/FEATURE_STORAGE/cfstore/source/cfstore_debug.h

```diff
@@ -28,6 +28,7 @@
   }while(0);
 
 #define noCFSTORE_DEBUG
+//#define CFSTORE_DEBUG
```

Comment:
This should be removed. Thanks. I'll remove it in a subsequent commit if that's OK.